### DISCUSSION TITLE
Major issue with relationships on External DB

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12228] [Help] [answered]
Hi i am using self host Budibase
I have Mysql database connected to it. I have tables that I have created relationship one to many. It doesn't matter if common (fk) column is number or text. After updating to 2.11.33 column info cannot be retrieved any more from linked table.
The relationship used to work fine on Budibase 2.8 and could retrieve information from linked tables. 
Is anyone facing same issue! THIS IS URGENT as it breaks my application.
Update to 2.12.1 does not solve the issue.
It seams that Provider filtering using a column from the linked table works, but display columns from linked table has stopped functioning both using formula columns and at Design.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @gmentzik,

There is an open issue for the formulas at the moment. Basically any external databases can't output relationship related information in formula columns at the moment. This issue can be found [here](https://github.com/Budibase/budibase/issues/11972).

**Reply by gmentzik** [reply]
Hi @ConorWebb96. Is there a plan to fix this soon enough. eg. within this week? Otherwise I will have to revert to 2.8 loosing all implementation changes I have done to the app we are developing the last couple of weeks. It doesn't look well when an update breaks an well defined existing functionality.

**Reply by ConorWebb96** [reply]
Hey @gmentzik,

I've got this bumped up to high priority after talking with a few of the engineers yesterday. Its likely not going to be fixed this week.

**Reply by gmentzik** [reply]
Hi @ConorWebb96 

Thank you for your response. 
I see various issues with Budibase 2.11, 2.12 with functionality that was working well enough before. I have reverted back to a backup I had from 2.8.10 which according to my experience was pretty stable and with clean admin layout. Thankfully I had not performed many changes to the app since that backup. I think I will wait a while monitoring bugs list before updating again.
